integrationTests/json/vm_test.go

```diff
@@ -82,10 +82,12 @@ func TestRustPayableFeaturesLatest(t *testing.T) {
 }
 
 func TestRustAsyncCalls(t *testing.T) {
+	// TODO fix excluded tests and include them back
```

Comment:
this TODO still remains ?

Reply:
Still remains, but only for some of the excluded tests. I have restored 3 of them, because now they pass.